chia/full_node/mempool_manager.py

```diff
@@ -82,6 +84,69 @@ def validate_clvm_and_signature(
     return None, bytes(result), new_cache_entries
 
 
+def compute_spend_cost(
+    puzzle_reveal: SerializedProgram, solution: SerializedProgram, maximum_cost: uint64
+) -> Optional[uint64]:
+    try:
+        cost, _ = puzzle_reveal.run_with_cost(maximum_cost, solution)
```

Comment:
you actually get the `CREATE_COIN` conditions out of running this here, so you could subtract these from the additions when you remove the duplicate